test/unit/controllers/components/create-model-form.js

```diff
@@ -43,9 +43,9 @@ describe('ModelFormController', () => {
                 })
             );
 
-            ModelFormController.createModel('OIUIO').then(
-                assert.fail
-            ).catch((e) => {
+            return ModelFormController.createModel('OIUIO').then(() => {
+                assert.fail;
```

Comment:
The fail is there in the case that for some reason the promise does resolve correctly we still get a notification that the test fails.